runtime/lua/vim/_system.lua

```diff
@@ -245,6 +245,10 @@ local function spawn(cmd, opts, on_exit, on_error)
   local handle, pid_or_err = uv.spawn(cmd, opts, on_exit)
```

Comment:
Should we change the return type of `uv.spawn` to indicate that the 2nd return value can be a string?

Reply:
yeah but I think that's waiting for https://github.com/luvit/luv/pull/762 to update the (rather outdated) type annotations?